test/adapter/json_api/has_many_test.rb

```diff
@@ -4,6 +4,24 @@ module ActiveModelSerializers
   module Adapter
     class JsonApi
       class HasManyTest < ActiveSupport::TestCase
+        module Serializers
+          Post = Class.new(ActiveModel::Serializer) do
```

Comment:
I'd prefer defining these classes like this `class Post < ActiveModel::Serializer`